site/docs/getting-started/Dialects.tsx

```diff
@@ -55,9 +60,10 @@ export function Dialects(props: DialectsProps) {
         it. This requires a <code>Dialect</code> implementation.
         <br />
         <br />
-        There are 3 built-in Node.js dialects for PostgreSQL, MySQL and SQLite.
-        Additionally, the community has implemented several dialects to choose
-        from. Find out more at <Link to="/docs/dialects">"Dialects"</Link>.
+        There are 4 built-in Node.js dialects for PostgreSQL, MySQL, SQLite, and
+        MSSQL. Additionally, the community has implemented several dialects to
+        choose from. Find out more at{' '}
```

Comment:
- Let's use the official long name and abbreviation.
- Let's place it before SQLite as it is still more popular.

```suggestion
        There are 4 built-in dialects for PostgreSQL, MySQL, Microsoft SQL Server 
        (MSSQL), and SQLite. Additionally, the community has implemented several
        dialects to choose from. Find out more at{' '}
```